plugin/loader/preload_list

```diff
@@ -8,3 +8,4 @@ ipldgit github.com/ipfs/go-ipfs/plugin/plugins/git 0
 badgerds github.com/ipfs/go-ipfs/plugin/plugins/badgerds 0
 flatfs github.com/ipfs/go-ipfs/plugin/plugins/flatfs 0
 levelds github.com/ipfs/go-ipfs/plugin/plugins/levelds 0
+postgresds github.com/ipfs/go-ipfs/plugin/plugins/postgresds 0
```

Comment:
While it may be useful for quite a few people, I wouldn't preload this plugin by default (leaving commented out is fine)

Reply:
Thank you for all the help, magik6k!